Automattic/wp-calypso
# Refactor migration onboarding steps to use new `FlowCard` component

Built on top of https://github.com/Automattic/wp-calypso/pull/92347


## Proposed Changes

* Apply the design to the cards on the "What do you want to do?" step. Now, the cards should follow the same pattern. We also introduced the changes to the "How to migrate" step on: https://github.com/Automattic/wp-calypso/pull/92240
* We also updated the header to use the Stepper default prop instead of creating the header ourselves.
* Please merge this at the same time with https://github.com/Automattic/wp-calypso/pull/92240 to avoid inconsistencies. 

#### Before

<img width="787" alt="Captura de Tela 2024-07-04 às 17 22 50" src="https://github.com/Automattic/wp-calypso/assets/1234758/b4fadd96-08dc-488b-b701-ad433469aa46">

#### After

<img width="708" alt="Captura de Tela 2024-07-04 às 17 20 18" src="https://github.com/Automattic/wp-calypso/assets/1234758/357ff935-7269-4fc2-8825-ee691a5e6353">

## Testing Instructions

* Apply this PR to your local environment or use the calypso live link below
* Use the migration onboarding flow (`/start`)
* Once on the `import-or-migrate` step, you should see the new components:

<img width="708" alt="Captura de Tela 2024-07-04 às 17 20 18" src="https://github.com/Automattic/wp-calypso/assets/1234758/ea08e65d-b17f-4a1b-afec-9819d02c587b">

* Verify navigation still works as intended
